x-pack/plugins/lens/public/datatable_visualization/components/cell_value.tsx

```diff
@@ -75,7 +76,7 @@ export const createGridCell = (
          */
         dangerouslySetInnerHTML={{ __html: content }} // eslint-disable-line react/no-danger
         data-test-subj="lnsTableCellContent"
-        className={`lnsTableCell ${alignmentClassName}`}
+        className={`${fitRowToContent ? '' : 'lnsTableCell'} ${alignmentClassName}`}
```

Comment:
nitpick: could you evaluate the ternary in the closure, together with the `IS_DARK_THEME`?

Reply:
done